analysis-pipeline/predictability/fit.conseq

```diff
@@ -0,0 +1,155 @@
+include "predictability_inputs.conseq"
+
+# Three Steps:
+# 1. Generate a daintree input config file for each model and screen
+# 2. Run the model fitting
+# 3. Combine the output config files
+
+rule process_model_config:
+    inputs:
+        # Model config yaml file
+        model_config=fileref("model-config.yaml"),
+
+        # Two target matrices
+        crispr_gene_effect={"type": "target_matrix", "label": "crispr_gene_effect"},
+        rnai={"type": "target_matrix", "label": "rnai"},
+
+        # Twelve features
+        lineage={"type": "feature", "label": "lineage"},
+        crispr_confounder={"type": "feature", "label": "crispr_confounder"},
+        rnai_confounder={"type": "feature", "label": "rnai_confounder"},
+        driver_events={"type": "feature", "label": "driver_events"},
+        armlevel_cna={"type": "feature", "label": "armlevel_cna"},
+        cytoband_cn={"type": "feature", "label": "cytoband_cn"},
+        genetic_signature={"type": "feature", "label": "genetic_signature"},
+        mutations_hotspot={"type": "feature", "label": "mutations_hotspot"},
+        mutations_damaging={"type": "feature", "label": "mutations_damaging"},
+        gene_cn={"type": "feature", "label": "gene_cn"},
+        loh={"type": "feature", "label": "loh"},
+        rnaseq={"type": "feature", "label": "rnaseq"},
+
+        # Script to generate daintree input config file
+        script=fileref("scripts/generate_daintree_input_configs.py"),
+
+    run "python" with """
+    import json
+
+    config_dict = {}
+
+    for key, value in {{inputs}}.items():
```

Comment:
I'm not really sure why this loop is needed to construct the dictionary.

Can't we just assign the dict to `config_dict`? 

That is to say:
```
config_dict = {{inputs}}
```